src/main/java/io/github/azagniotov/metrics/reporter/cloudwatch/CloudWatchReporter.java

```diff
@@ -339,6 +341,7 @@ private void stageMetricDatum(final boolean metricConfigured,
                     .withValue(cleanMetricValue(metricValue))
                     .withMetricName(metricName)
                     .withDimensions(dimensions)
+                    .withStorageResolution(highResolution ? 1 : 60)
```

Comment:
What do `1` & `60` mean here? Could you extract them into constants?

Reply:
I guess they mean something like "one second resolution" or "sixty seconds/one minute resolution"... AWS doesn't give much away;

> The PutMetricData function now accepts an optional StorageResolution parameter. Set this parameter to 1 to publish high-resolution metrics; omit it (or set it to 60) to publish at standard 1-minute resolution.

Since this is `CloudWatchReporter` internals, I thought that abstracting them using `Builder.withHighResolution()` would be enough :-)

Reply:
ok - got rid of the secret numbers